src/bootstrap-theme/_forms.scss

```diff
@@ -0,0 +1,10 @@
+$radio-checked-marker-radius: $fhi-core-px * 7; // the checked marker will be 14px in diameter
```

Comment:
Since this variables isn't local they should be fhi-core-variables.
And no files should be imported that doesn't correspond with https://github.com/twbs/bootstrap/blob/main/scss/_forms.scss